www/install.rst

```diff
@@ -84,9 +84,12 @@ For Python 3.5 with `Macports <https://www.macports.org>`_ execute this command
 
 .. rubric:: Homebrew
 
-You can install NumPy, SciPy, and Matplotlib, with::
+Homebrew is not the correct way to install NumPy. We reccomend
```

Comment:
I suggest: "Homebrew no longer packages NumPy or other packages of the scientific Python stack. If you're a Homebrew user, installing only Python with Homebrew and the rest with Pip is the recommended method::"